helicopyter.py

```diff
@@ -127,29 +130,40 @@ def multisynth(all_or_conas_or_paths: Iterable[str]) -> None:
             cona = path_to_check.parent.parent.name
         else:
             cona = cona_or_path
-        relative_path = f'deploys/{cona}/terraform/main.tf.json'
+        relative_path = (
+            f'deploys/{cona}/terraform/main.tf{"" if hashicorp_configuration_language else ".json"}'
+        )
         print(f'Generating {relative_path}')
         module = import_module(f'deploys.{cona}.terraform.main')
         stack = HeliStack(cona)
         module.synth(stack)
-        dictionary = stack.to_terraform()
-        dictionary['//']['AUTOGENERATED'] = 'by helicopyter'
-        if stack.imports:
-            dictionary['import'] = [
-                {'to': resource, 'id': import_id} for resource, import_id in stack.imports.items()
-            ]
-        with (Path(__file__).parent / relative_path).open('w') as output:
-            dump(dictionary, output, indent=4, sort_keys=True)
-            output.write('\n')
+        if hashicorp_configuration_language:
+            unformatted = stack.to_hcl_terraform()['hcl'].encode()
+            formatted = check_output(['terraform', 'fmt', '-'], input=unformatted).decode()  # noqa: S603
+            (Path(__file__).parent / relative_path).write_text(
+                '# AUTOGENERATED by helicopyter\n\n' + formatted
+            )
+        else:
+            dictionary = stack.to_terraform()
+            dictionary['//']['AUTOGENERATED'] = 'by helicopyter'
+            if stack.imports:
+                dictionary['import'] = [
+                    {'to': resource, 'id': import_id}
+                    for resource, import_id in stack.imports.items()
+                ]
+            with (Path(__file__).parent / relative_path).open('w') as output:
+                dump(dictionary, output, indent=4, sort_keys=True)
+                output.write('\n')
 
 
 class Parameters(Tap):
-    conas: list[str]
+    conas: list[str]  # space-separated COdeNAmes
+    hashicorp_configuration_language: bool = False
 
     def configure(self) -> None:  # noqa: D102
-        self.add_argument('conas', help='COdeNAmes')
+        self.add_argument('conas')  # Make this a positional argument
```

Comment:
Why get rid of `help`? Imo you should keep the `help`, as it explained what `conas` means

Reply:
Moved not gone
> Single line and/or multiline comments which appear after the argument are automatically parsed into the help string provided when running python main.py -h. The type and default values of arguments are also provided in the help string. https://github.com/swansonk14/typed-argument-parser?tab=readme-ov-file#tap-help